backend/pick-git/src/main/java/com/woowacourse/pickgit/post/infrastructure/GithubExtractor.java

```diff
@@ -0,0 +1,39 @@
+package com.woowacourse.pickgit.post.infrastructure;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.woowacourse.pickgit.post.domain.PlatformExtractor;
+import com.woowacourse.pickgit.post.infrastructure.dto.RepositoryResponse;
+import java.util.List;
+import org.springframework.stereotype.Component;
+
+@Component
+public class GithubExtractor implements PlatformExtractor {
+
+    private final ObjectMapper objectMapper;
+    private final PlatformApiRequester platformApiRequester;
+
+    public GithubExtractor(
+        ObjectMapper objectMapper,
+        PlatformApiRequester platformApiRequester) {
+        this.objectMapper = objectMapper;
+        this.platformApiRequester = platformApiRequester;
+    }
```

Comment:
```suggestion
    public GithubExtractor(
        ObjectMapper objectMapper,
        PlatformApiRequester platformApiRequester) 
    {
        this.objectMapper = objectMapper;
        this.platformApiRequester = platformApiRequester;
    }
```

이런식의 컨벤션은 로직이랑 파라메터를 구분하기 힘들어서 가독성이 좋지 않아요.
구글 컨벤션은 제가 제안한 쪽의 컨벤션을 유도한는 것 같구 다른 소스를 많이 보면 이런식으로 정리를 많이 하더라구요 어떤가요?

Reply:
opt + cmd + L 했을 때 제가 작성한 것과 같이 formatting이 돼요! 그래서 현재 코드처럼 정리했습니다.
개인적으로 `{`이 `)` 다음에 바로 오는 걸 선호하는 편인데, 이 부분은 팀원들과 한번 간단하게 논의해볼까요?